thefuck/corrector.py

```diff
@@ -24,7 +24,16 @@ def get_categories(command):
     """
     CATEGORIES = {'git': {'git'},
                   'brew': {'brew'},
-                  'django': {'manage.py', 'migrate'}}
+                  'django': {'manage.py', 'migrate'},
+                  'grep': {'grep'},
+                  'grep': {'egrep'},
+                  'cargo': {'cargo'},
+                  'maven': {'mvn'},
+                  'rm': {'rm'},
+                  'pacman': {'pacman'},
+                  'pacman': {'yaourt'},
+                  'cd': {'cd'},
```

Comment:
cd parent needs to be included even if cd is not found in the script. The cd parent rule changes cd.. to cd ..